src/SFA.DAS.EmployerAccounts.Web/Views/Home/Unsubscribed.cshtml

```diff
@@ -0,0 +1,17 @@
+﻿@{ViewBag.PageID = "unsubscibed-from-emails"; }
+@{ViewBag.Title = "Unsubscribed from emails"; }
+@{ViewBag.MetaDesc = "Unsubscribed from emails"; }
+@{ViewBag.HideNav = true; }
+
+@{
+    ViewBag.GaData.Vpv = "/unsubscibed-from-emails";
+}
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <div class="panel">
+            <h1 class="heading-large">You are now unsubscribed</h1>
+            <p>You have now been unsubscribed and will no longer receive invitation emails from this training provider.</p>
+        </div>
+    </div>
+</div>
```

Comment:
old styling pre-CDN, is that fine?

Reply:
It'll need updating.. I can either get Darren to do it now or after it's merged? This won't be visible until after the release of provider led